### ISSUE TITLE
Difference between this and CryptoNote

### ISSUE THREAD

**Issue by k-ool** [bcndev/bytecoin#121]
What is the main difference from this and the CryptoNote fork? CryptoNote seems to be  more outdated. Should I use this over CryptoNote?

**Comment by ghost** [comment]
Yes. CryptoNote is outdated since years! Karbo coin can be still be used :D
